contracts/contracts/vault/VaultAdmin.sol

```diff
@@ -449,7 +451,14 @@ contract VaultAdmin is VaultStorage {
     {
         IStrategy strategy = IStrategy(_strategyAddr);
         _harvest(address(strategy));
-        return _swap(strategy.rewardTokenAddress());
+        address[] memory rewardTokens = strategy.getRewardTokenAddresses();
+        uint256[] memory liquidationLimits = strategy.getRewardLiquidationLimits();
+        require(rewardTokens.length == liquidationLimits.length, "Reward token array and liquidation limit array must be of the same size");
+
+        for (uint256 i = 0; i < rewardTokens.length; i++) {
+            // TODO: is it necessary we return the swap results here?
```

Comment:
No, I don't think we need to return the swap results.

Reply:
Cool changing so that function doesn't have a return type.